impl/src/planning/subplanners.rs

```diff
@@ -245,12 +245,15 @@ impl<'planner> WorkspaceSubplanner<'planner> {
     }
 
     // Additionally, the binary dependencies need to have their checksums added as well
```

Comment:
can you update the comment to be specific to remote, and mention why vendored shouldn't do this?